src/Illuminate/Contracts/Auth/Guard.php

```diff
@@ -51,7 +51,7 @@ public function hasUser();
      * Set the current user.
      *
      * @param  \Illuminate\Contracts\Auth\Authenticatable  $user
-     * @return void
+     * @return $this
```

Comment:
This is a breaking change

Reply:
In the **TokenGuard** class, a trait called **GuardHelpers** is used ,and in that there is a method called setUser. Note the output of this method

![GuardHelpers](https://user-images.githubusercontent.com/41255155/163710693-4b06001f-c72d-41b6-997a-35b60e3ab959.png)

- Illuminate\Auth\GuardHelpers (line 91)

Reply:
The `setUser` methods that implement this contract should be `void`. We don't use the return anywhere in the framework.